lib/block_view_balance_entry.go

```diff
@@ -165,17 +167,38 @@ func (bav *UtxoView) GetHolders(pkid *PKID, fetchProfiles bool, isDAOCoin bool)
 		}
 	}
 
+	// Fetch all the locked entries if we're dealing with a dao coin
```

Comment:
Please break this out into a separate function. Fetching all the holders for unlocked is already pretty slow since we have to get all the profiles as well. we can call both functions in backend if you need it, but it's a lot of unnecessary overhead for existing usages. This is especially confusing when this function is used to fetch creator coin holdings.